build-and-test.ps1

```diff
@@ -1,7 +1,7 @@
 [cmdletbinding()]
 param(
     [switch]$UseImageCache,
-    [string]$Filter,
```

Comment:
Upon looking at this a second time, I don't think the parameter names consistent yet.  The use of the `Filter` suffix is inconsistent.  All parameters are a type of filter - that is how they are named in the ImageTests.

Reply:
Yes, naming it `Version` would be sufficient.

Reply:
What do you think about adding the `Filter` suffix to all params?  That would be more descriptive.

Reply:
Yes, it would be more descriptive. I'll update.